lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -206,6 +242,10 @@
         "cid": {
           "type": "cid-link",
           "description": "For creates and updates, the new record CID. For deletions, null."
+        },
+        "prev": {
+          "type": "cid-link",
+          "description": "For updates and deletes, the previous record CID (required). For creations, null."
```

Comment:
This mentions that `prev` is `null` for creations.  Two things here to poke at: a. it's not currently marked as nullable, b. marking it as nullable would make it optional-nullable, which is a pattern we want to try to avoid.  I don't think there's a perfect path forward, but my inclination is to leave the lexicon as-is and omit it for creation ops.

Reply:
Ah yep, meant for these to stay "optional" not "nullable". Changed the wording to "For creations, field should not be defined."

Also updated from "(required)" to "(required for inductive firehose)".